server/views/referrals/courseParticipations/details/show.njk

```diff
@@ -61,16 +66,29 @@
   classes: "govuk-!-width-one-third",
   label: {
     text: "Enter the year started (if known)"
-  }
+  },
+  errorMessage: errors.messages.yearStarted,
+  value: formValues.outcome.yearStarted
 }) }}
 {% endset %}
 
 {% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if errors.list.length %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: errors.list
+        }) }}
+      {% endif %}
+    </div>
+  </div>
+
   <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <form action="#" method="post">
+      <form action="{{ referPaths.programmeHistory.details.update({ courseParticipationId: courseParticipationId, referralId: referralId }) }}?_method=PUT" method="post">
```

Comment:
Is this in the right commit? It's not really related to showing errors and or populating the form with existing values

Reply:
Yeah probably not.  Was actually going to change it to how you've done it and set `action` in the controller.